src/styles/components/header-bar/styles.less

```diff
@@ -9,8 +9,9 @@
     height: @header-bar-height;
     padding-left: @header-bar-horizontal-padding;
     padding-right: @header-bar-horizontal-padding;
+    text-rendering: optimizeLegibility;
     width: 100%;
-    z-index: 999;
+    z-index: 9999;
```

Comment:
An additional "flash" was happening on the dropdown trigger text because the dropdown menu panel would briefly overlap the cursor when it animates in, which would break the trigger's hover styling

Reply:
Nice! thanks for the explanation.